lib/chef/knife/rackspace_server_create.rb

```diff
@@ -434,15 +441,26 @@ def run
           msg_pair('rax_service_level_automation', server.metadata.all['rax_service_level_automation'])
           Chef::Application.fatal! 'Server didn\'t finish on time'
         end
+
         msg_pair("Metadata", server.metadata)
-        if(networks && Chef::Config[:knife][:rackspace_networks])
-          msg_pair("Networks", Chef::Config[:knife][:rackspace_networks].sort.join(', '))
-        end
 
         print "\n#{ui.color("Waiting server", :magenta)}"
 
         puts("\n")
 
+        if Chef::Config[:knife][:rackconnect_v3_network_id]
+          print "\n#{ui.color("Setting up RackconnectV3 network and IPs", :magenta)}"
+          setup_rackconnect_network!(server)
+          while server.ipv4_address == ""
+            server.reload
+            sleep 5
```

Comment:
Will this loop ever exit if, for example, the server goes into an error state/build failure? Anything that introduces an infinite loop feels like a potentially painful change in behavior.